x-pack/filebeat/input/s3/input.go

```diff
@@ -411,7 +411,7 @@ func (p *s3Input) createEventsFromS3Info(svc s3iface.ClientAPI, info s3Info, s3C
 			// If the SDK can determine the request or retry delay was canceled
 			// by a context the ErrCodeRequestCanceled error will be returned.
 			if awsErr.Code() == awssdk.ErrCodeRequestCanceled {
-				err = errors.Wrap(err, "GetObject request canceled")
+				err = errors.Wrap(err, "S3 GetObjectRequest canceled")
```

Comment:
Typo in cancelled, missing `l`.

```suggestion
				err = errors.Wrap(err, "S3 GetObjectRequest cancelled")
```

Reply:
In AWS SDK, their error code is `ErrCodeRequestCanceled` for `RequestCanceled`. I'm gonna keep it consistent here to use canceled 😄 Apparently both spelling are correct?!